src/http-gateways/trustless-gateway.md

```diff
@@ -301,6 +302,112 @@ of their presence in the DAG or the value assigned to the "dups" parameter, as
 the raw data is already present in the parent block that links to the identity
 CID.
 
+## `meta` (content type parameter)
+
+The `meta` parameter allows clients to request the server to include additional metadata about the CAR along with the response body.
+
+The value of this parameter includes both the location where the metadata is given (e.g. `eof`) as well as the type of data received (e.g. `json`) separated by a `+`, to give a value such as `meta=eof+json`
+
+When the location parameter is set to `eof`, which is currently the only supported value, the server SHOULD respond with the following response body:
+
+```
+<Response body as CARv1 stream> <0x00 byte> <Metadata>
+```
```

Comment:
It looks like the template used by the website cannot handle `<>` characters correctly :( Is it something worth fixing or should you change this PR to use different characters?

E.g. follow the IPLD style:

```
Response body as CARv1 stream | 0x00 byt | Metadata
```

Reply:
<img width="831" alt="Screenshot 2023-10-18 at 16 16 38" src="https://github.com/ipfs/specs/assets/1140553/a3c2ec40-771b-4e66-877d-07e9b972990f">

Reply:
This is a bug in website generator, not blocking this PR. 
Fix tracked in https://github.com/ipfs/spec-generator/issues/19

Reply:
For now just use 
1```suggestion
```
[Response body as CARv1 stream] [0x00 byte] [Metadata]
```
````

Reply:
I see that https://github.com/ipfs/spec-generator/issues/19 was fixed, I will stick to `<>`.